common/dynamicconfig/constants_test.go

```diff
@@ -0,0 +1,319 @@
+// The MIT License (MIT)
+
+// Copyright (c) 2017-2020 Uber Technologies Inc.
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package dynamicconfig
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/uber/cadence/common"
+)
+
+type constantSuite struct {
+	suite.Suite
+}
+
+func TestConstantSuite(t *testing.T) {
+	suite.Run(t, new(constantSuite))
+}
+
+func (s *constantSuite) TestListAllProductionKeys() {
+	//check if we given enough capacity
+	s.GreaterOrEqual(len(IntKeys)+len(BoolKeys)+len(FloatKeys)+len(StringKeys)+len(DurationKeys)+len(MapKeys), len(ListAllProductionKeys()))
+
+}
+
+func (s *constantSuite) TestIntKey() {
+	testIntKeys := map[string]struct {
+		key                  IntKey
+		expectedString       string
+		expectedDefaultValue int
+		expectedDescription  string
+		expectedFilters      []Filter
+	}{
+		"TestGetIntPropertyKey": {
```

Comment:
are you planning to cover the rest of the keys in follow up PR?

Reply:
Not really as variable definition is not included in code coverage and including all keys will be essentially parsed structs in constants.go.